app/src/main/res/layout/row_loan_account.xml

```diff
@@ -1,46 +1,108 @@
+            <TextView
+                android:id="@+id/tv_clientLoanAccountNumber"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:ellipsize="end"
+                android:lines="1"
+                android:text="@string/medium_text"
+                android:textColor="@color/black"
+                android:textSize="@dimen/text_medium" />
 
+            <TextView
+                android:id="@+id/tv_loanAccountProductName"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/small_text"
+                android:textColor="@color/gray_dark"
+                android:textSize="@dimen/text_small" />
+
+        </LinearLayout>
+    </LinearLayout>
     <LinearLayout
-        android:layout_width="0dp"
+        android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_weight="1"
-        android:gravity="center_vertical"
-        android:orientation="vertical"
-        android:paddingStart="@dimen/row_padding_left">
-
-        <TextView
-            android:id="@+id/tv_clientLoanAccountNumber"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:ellipsize="end"
-            android:lines="1"
-            android:text="@string/medium_text"
-            android:textColor="@color/black"
-            android:textSize="@dimen/text_medium" />
-
-        <TextView
-            android:id="@+id/tv_loanAccountProductName"
-            android:layout_width="wrap_content"
+        android:id="@+id/ll_account_detail"
+        android:visibility="gone"
+        android:orientation="vertical">
+        <View
+            android:layout_width="match_parent"
+            android:layout_height="1dp"
+            android:background="@color/gray_light"/>
+        <LinearLayout
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="@string/small_text"
-            android:textColor="@color/gray_dark"
-            android:textSize="@dimen/text_small" />
+            android:gravity="center_vertical"
+            android:orientation="horizontal"
+            android:layout_marginTop="@dimen/default_margin"
+            android:layout_marginBottom="@dimen/default_margin">
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/balance"
+                android:textColor="@color/gray_dark"
+                android:textSize="@dimen/text_small"/>
+            <TextView
+                android:id="@+id/tv_account_balance"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="20000 $"
```

Comment:
Don't use hardcoded string, define string in strings.xml.
example: ``<string "amount">Amount</string>``